emu/processes/wps_test_ncml.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from pywps import Process
+from pywps import ComplexInput, ComplexOutput, FORMATS, Format
+from pywps.inout.basic import SOURCE_TYPE
+from pywps.validator.mode import MODE
+from pywps.app.Common import Metadata
+from pathlib import Path
+
+from netCDF4 import Dataset
+
+import logging
+LOGGER = logging.getLogger("PYWPS")
+
+
+class NcMLTest(Process):
```

Comment:
Where is the difference to NcMLAgg?

Reply:
None, this is a left-over from a file rename...